api-service/crowdflower/crowdflower.js

```diff
@@ -0,0 +1,36 @@
+<iframe
+  src="https://crowdsource-demo-dot-wikidetox.appspot.com/?embedded=true#{{client_job_key}}/{{question_id}}"
+  width="400"
+  height="425"
+  frameborder="0"
+  scrolling="no"
+  marginheight="0"
+  marginwidth="0"
+  >
+</iframe>
+
+<cml:hidden
+  default="test"
+  question-label="Sample text field:"
+  validates="required"
+  value="test"
+  name="question_comments">
+</cml:hidden>
+
+<script>
+require(['jquery-noconflict'], function($) {
+    //Ensure MooTools is where it must be
+    Window.implement('$', function(el, nc){
+	return document.id(el, nc, this.document);
+    });
+
+    var $ = window.jQuery;
+
+    window.onmessage = function(e){
+	console.log('got message', e.data);
+	var data = JSON.parse(e.data);
+
+	$('.question_comments')[0].value = e.data;
```

Comment:
Add a TODO here to add a function that should validate the data as being an acceptable format to store in value / to test if we need such a function (this is regarding the bug I mentioned during the meeting, where crowdflower will silently fail to record responses with invalid characters in them).

Reply:
Will do